scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.adoc

```diff
@@ -1,21 +1,18 @@
 :_content-type: ASSEMBLY
-// CNF-1500 assembly
-[id="ztp-deploying-disconnected"]
-= Deploying distributed units at scale in a disconnected environment
+[id="ztp-deploying-far-edge-clusters-at-scale"]
+= Scaling cluster deployments at the far edge
 include::_attributes/common-attributes.adoc[]
-:context: ztp-deploying-disconnected
+:context: ztp-deploying-far-edge-clusters-at-scale
 
 toc::[]
 
-Use zero touch provisioning (ZTP) to provision distributed units at new edge sites in a disconnected environment. The workflow starts when the site is connected to the network and ends with the CNF workload deployed and running on the site nodes.
+Telco edge computing presents complex challenges with managing large numbers of clusters across many geographically displaced locations.
 
-include::modules/ztp-provisioning-edge-sites-at-scale.adoc[leveloffset=+1]
+Use zero touch provisioning (ZTP) and GitOps to provision clusters at new sites at the very far edge of the network. The workflow starts when the site is connected to the network and ends with the cloud-native network function (CNF) workloads deployed and running on the site nodes.
```

Comment:
I don't agree with the latter part here
"ends with the cloud-native network function (CNF) workloads deployed and running on the site nodes."

- The ZTP workload is used for day-2 operations, i.e. rolling out changes and updates across the edge sites. I.e. it hardly stops once the workload is deployed
- Orchestrating the workload is usually the domain of a workload orchestrator.
Although this of course might be GitOps driven, it is usually not the domain of the tooling described in this document